src/components/app/App.jsx

```diff
@@ -141,6 +141,43 @@ export const App = () => {
 
     }, [] );
 
+    useEffect( () => {
```

Comment:
This is not a good place to hook in the functionality. It sounds more like a job for `ioManager`.

Reply:
Updated and moved key handling to ioManager.